coordinator/src/node.rs

```diff
@@ -48,21 +48,21 @@ pub struct Node {
 }
 
 impl Node {
-    /// Returns true or false, whether we can find an useable channel with the provided trader.
+    /// Returns true or false, whether we can find an usable channel with the provided trader.
```

Comment:
It's not a typo[^1] 🙈 But I'm happy with the patch since it aligns it with LDK's terminology.

[^1]: It's a less common variant according to [Merriam Webster](https://www.merriam-webster.com/dictionary/usable).

Reply:
same same. tomato tomato. :)

Reply:
BTW: American dictionaries don't convince me ;D it's not the variant of English that I'm using in my daily speech.